src/services/leafletWatchHelpers.js

```diff
@@ -20,7 +20,17 @@ angular.module("leaflet-directive")
   to set watching to once and its watch depth.
   */
   var _maybeWatch = function(scope, thingToWatchStr, watchOptions, initCb){
-      return _maybe(scope, '$watch', thingToWatchStr, watchOptions, initCb);
+      var localWatchOptions, watchMethod;
```

Comment:
@elesdoar thoughts here at all? If we change to just watchOptions below then we will bump to 0.10.X